billing/src/main/java/com/appcoins/wallet/billing/common/response/TransactionFee.kt

```diff
@@ -0,0 +1,15 @@
+package com.appcoins.wallet.billing.common.response
+
+import java.math.BigDecimal
+
+data class TransactionFee(val type: Type, val cost: Cost?) {
+
+  enum class Type {
+    EXACT, UNKNOWN
+  }
+
+  data class Cost(val value: BigDecimal, val currency: String)
+}
```

Comment:
Can't we use PaymentMethodFee?

Reply:
That class is not in this module unless we extract it